core/templates/dev/head/pages/exploration_player/PlayerConstants.js

```diff
@@ -22,6 +22,8 @@ oppia.constant('TWO_CARD_THRESHOLD_PX', 960);
 
 oppia.constant('CONTINUE_BUTTON_FOCUS_LABEL', 'continueButton');
 
+oppia.constant('WAIT_FOR_HINT_MSEC', 30000);
```

Comment:
Add a single line comment above this to explain what it represents. I get that there's some waiting going on for 30 secs but I don't understand when it starts.